edx_proctoring/tests/test_student_view.py

```diff
@@ -161,29 +184,47 @@ def test_verification_status(self, verification_status, expected_message):
         This test asserts that the correct id verification message is shown
         to the user when they choose to take a proctored exam.
         """
-
         self._create_unstarted_exam_attempt()
+        rendered_response = self.render_proctored_exam({
+            'verification_status': verification_status,
+        })
+        self.assertIn(expected_message, rendered_response)
 
-        rendered_response = get_student_view(
-            user_id=self.user_id,
-            course_id=self.course_id,
-            content_id=self.content_id,
-            context={
-                'is_proctored': True,
-                'display_name': self.exam_name,
-                'default_time_limit_mins': 90,
-                'is_practice_exam': False,
-                'credit_state': {
-                    'enrollment_mode': 'verified',
-                    'credit_requirement_status': [
-                    ]
-                },
-                'verification_status': verification_status,
-                'verification_url': '/reverify',
-            }
-        )
+    def test_proctored_only_entrance(self):
+        """
+        This test verifies that learners are not given the option to enter an unproctored exam
```

Comment:
For comments, even thought it is longer, I think referring to a "proctored exam without proctoring" is more accurate than an "unproctored" exam, which I would interpret as a "timed exam".